packages/e2e/local-network/scripts/mint-handles.sh

```diff
@@ -51,3 +51,45 @@ cardano-cli transaction sign \
   --out-file handle-tx.signed
 
 cardano-cli transaction submit --testnet-magic 888 --tx-file handle-tx.signed
+
+sleep 10
+# CIP-68 Handle
+utxo=$(cardano-cli query utxo --address "$addr" --testnet-magic 888 | awk 'NR == 3 {printf("%s#%s", $1, $2)}')
+cat >network-files/utxo-keys/handles68-datum.json <<EOL
+{"constructor": 0,
+  "fields": [{"map": [
+    {"k": {"bytes": "636f7265"}, "v": {"map": [
+      {"k": {"bytes": "6f67"}, "v": {"int": 0}},
+      {"k": {"bytes": "707265666978"}, "v": {"bytes": "24"}},
+      {"k": {"bytes": "76657273696f6e"}, "v": {"int": 0}},
+      {"k": {"bytes": "7465726d736f66757365"}, "v": {"bytes": "68747470733a2f2f63617264616e6f666f756e646174696f6e2e6f72672f656e2f7465726d732d616e642d636f6e646974696f6e732f"}},
+      {"k": {"bytes": "68616e646c65456e636f64696e67"}, "v": {"bytes": "7574662d38"}}
+    ]}},
+    {"k": {"bytes": "6e616d65"}, "v": {"bytes": "283130302968616e646c653638"}},
+    {"k": {"bytes": "696d616765"}, "v": {"bytes": "697066733a2f2f736f6d652d68617368"}},
+    {"k": {"bytes": "77656273697465"}, "v": {"bytes": "68747470733a2f2f63617264616e6f2e6f72672f"}},
+    {"k": {"bytes": "6465736372697074696f6e"}, "v": {"bytes": "5468652048616e646c65205374616e64617264"}},
+    {"k": {"bytes": "6175676d656e746174696f6e73"}, "v": {"list": []}}]},
+    {"int": 1}
```

Comment:
Seems like we're missing the 3rd field:
```
The version number is at the second field of this constructor. The third field allows for arbitrary plutus data.
```
```
  ; Custom user defined plutus data.
    ; Setting data is optional, but the field is required
    ; and needs to be at least Unit/Void: #6.121([])
    extra = plutus_data
```

Reply:
added in [e3bea15](https://github.com/input-output-hk/cardano-js-sdk/pull/807/commits/e3bea157c1d241db8de7a2505d33d5c1b579e153)